public/app/assetlibrary/assetlibrary_item/assetlibrary_item.html

```diff
@@ -0,0 +1 @@
+<h1>Asset Library Item</h1>
```

Comment:
@nicolaasmatthijs GitHub seems to be complaining about adding an extra line break. If your editor supports it, I would definitely recommend adding a `.editorconfig` file.

https://github.com/ets-berkeley-edu/calcentral/blob/71c979a95cbdb81178f9e63929af39c269d9f672/.editorconfig#L7